impl_core/src/main/java/io/opencensus/implcore/trace/SpanImpl.java

```diff
@@ -313,6 +319,7 @@ public void addAnnotation(String description, Map<String, AttributeValue> attrib
 
   @Override
   public void addAnnotation(Annotation annotation) {
+    Utils.checkNotNull(annotation, "annotation");
```

Comment:
There was already a null check later in the method, though I think that validating arguments at the start of the method is clearer.

Reply:
Oooh good catch. I'll remove the second one.